addons/website_forum/views/website_forum.xml

```diff
@@ -139,21 +151,23 @@
                     </a>
                 </li>
                 <li class="nav-item">
-                    <a t-attf-href="/forum/#{ slug(forum) }" t-attf-class="nav-link #{question_count and 'active'}">Topics</a>
+                    <a t-attf-href="/forum/#{ forum_slug }" t-attf-class="nav-link #{question_count and 'active'}">Topics</a>
                 </li>
                 <li class="nav-item">
                     <a t-attf-href="/profile/users?forum_origin=#{request.httprequest.path}"
                         t-attf-class="nav-link #{searches.get('users') and 'active'}">People</a>
                 </li>
-                <li class="nav-item">
-                    <a t-attf-href="/forum/#{ slug(forum) }/tag" t-attf-class="nav-link #{searches.get('tags') and 'active'}">Tags</a>
+                <li t-if="forum" class="nav-item">
+                    <a t-attf-href="/forum/#{ forum_slug }/tag" t-attf-class="nav-link #{searches.get('tags') and 'active'}">Tags</a>
                 </li>
                 <li class="nav-item">
-                    <a t-attf-href="/profile/ranks_badges?badge_category=forum&amp;url_origin=#{request.httprequest.path}&amp;name_origin=#{forum.name}"
+                    <a t-if="forum" t-attf-href="/profile/ranks_badges?badge_category=forum&amp;url_origin=#{request.httprequest.path}&amp;name_origin=#{forum.name}"
+                    t-attf-class="nav-link #{searches.get('badges') and 'active'}">Badges</a>
+                    <a t-else="" t-attf-href="/profile/ranks_badges?badge_category=forum&amp;url_origin=#{request.httprequest.path}"
                     t-attf-class="nav-link #{searches.get('badges') and 'active'}">Badges</a>
```

Comment:
you can merge both element I think (with just a condition to add the forum name)